x-pack/plugins/observability_shared/public/components/profiling/embeddables/embeddable_flamegraph.tsx

```diff
@@ -5,59 +5,13 @@
  * 2.0.
```

Comment:
would it make sense to change code owner for profiling directory to profiling team?